new/components/mgmtworker/config/cloudify-mgmtworker

```diff
@@ -5,7 +5,14 @@ CELERY_LOG_DIR="/var/log/cloudify/mgmtworker"
 RIEMANN_CONFIGS_DIR="/opt/riemann"
 MANAGEMENT_IP="{{ ctx.instance.host_ip }}"
 MANAGEMENT_USER="root"
-MANAGER_REST_PORT="8101"
+MANAGER_REST_PORT={{ ctx.instance.runtime_properties.rest_port }}
+SECURITY_REST_PROTOCOL={{ ctx.instance.runtime_properties.rest_protocol }}
+SECURITY_ENABLED={{ ctx.instance.runtime_properties.security_enabled }}
+CLOUDIFY_USERNAME={{ ctx.instance.runtime_properties.cloudify_username }}
+CLOUDIFY_PASSWORD={{ ctx.instance.runtime_properties.cloudify_password }}
+SSL_ENABLED={{ ctx.instance.runtime_properties.ssl_enabled }}
```

Comment:
not sure ssl_enabled is required, since port and protocol are already calculated (see preconfigure.sh too)